java/node/src/main/java/com/codeheadsystems/dstore/node/manager/TenantManager.java

```diff
@@ -103,7 +103,7 @@ public Tenant create(final String tenantId) {
   private Tenant buildTenant(final String tenantId) {
     LOGGER.debug("buildTenant({})", tenantId);
     return ImmutableTenant.builder()
-        .id(tenantId)
```

Comment:
Yeah, more I see this, I think tenantId should be the standard instead of id. Make it explicit